pkg/types/ccipocr3/generic_types.go

```diff
@@ -83,9 +83,25 @@ func (c ChainSelector) String() string {
 }
 
 type CCIPMsg struct {
```

Comment:
Pre-existing, but this type name is a bit redundant given the package: `ccipocr3.CCIPMsg`. Could be `ccipocr2.Msg`

Reply:
I'd prefer to keep it like this because we use some alias on imports and it might become `types.Msg`

Reply:
IMHO that is a very backwards way of addressing that problem